app/assets/stylesheets/search.scss

```diff
@@ -59,7 +60,53 @@
   border-radius: 0.5em !important;
   color: $my-black;
   padding-left: 0.7em !important;
+}
+
+.page-number {
+  display: inline;
+  ul li {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3